414. Third Maximum Number.py

```diff
@@ -0,0 +1,13 @@
+class Solution(object):
+    def thirdMax(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        sort_nums = list(set(nums))
```

Comment:
we can simplify this to `sort_nums = sorted(list(set(nums))`.
The difference beween `L.sort()` and `sorted(L)` is that `L.sort()` sorts the list in place, whereas `sorted(L)` creates and returns a sorted list from `L` (does not change L).